src/pyhepmc/view.py

```diff
@@ -151,3 +151,12 @@ def to_dot(
         )
 
     return d
+
+def save_event(evt: pyhepmc.GenEvent, name: str ="event.png"):
+    g = to_dot(evt)
+    if 'png' in name:
+        with open("event.png", "wb") as f:
+            f.write(g._repr_image_png())  # as png
+    elif 'svg' in name:
+        with open("event.svg", "w") as f:
+            f.write(g._repr_image_svg_xml())  # as svg (recommended)
```

Comment:
This looks good, but what if the user provides an extension that we cannot handle? We should raise an Exception then. matplotlib raises this one
```py
ValueError: Format 'foo' is not supported (supported formats: eps, jpeg, jpg, pdf, pgf, png, ps, raw, rgba, svg, svgz, tif, tiff, webp)
```
Instead of asking for the extension to be in the name, it is better to convert the argument to a [pathlib.PurePath](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath) first and then use the `suffix` attribute of the Path.